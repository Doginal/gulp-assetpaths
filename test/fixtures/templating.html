<img src="<?php echo $imagePath ?>" />
<img src="<%= obj.imagePath %>" />
<script src="<% scriptname %>" />
background: url(<% imagepath %>)
background: url(<?php echo $imagepath ?>)