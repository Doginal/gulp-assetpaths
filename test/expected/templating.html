<img src="https://www.newDomain.com/<?php echo $imagePath ?>" />
<img src="https://www.newDomain.com/<%= obj.imagePath %>" />
<script src="https://www.newDomain.com/<% scriptname %>" />
background: url(https://www.newDomain.com/<% imagepath %>)
background: url(https://www.newDomain.com/<?php echo $imagepath ?>)